poc/vdaf_prio3.py

```diff
@@ -483,6 +510,51 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+class Prio3SumVecWithMultiproof(Prio3SumVec):
+    ID = 0xFFFFFFFF  # To be set by `with_params`
+
+    # Operational parameters.
+    test_vec_name = 'Prio3SumVecWithMultiproof'
+
+    @staticmethod
+    def is_recommended(valid_cls,
+                       num_proofs: Unsigned,
+                       field: FftField) -> bool:
+        # To be confirmed
+        if field == Field64:
+            return num_proofs >= 2
+        elif field == Field96:
+            return num_proofs >= 2
+        elif field == Field128:
+            return num_proofs >= 1
+        return False
+
+    @staticmethod
+    def algorithm_id_from(num_proofs: Unsigned,
```

Comment:
What algorithm IDs shall we allocate for this variants? 

Let $N$ be the next available algorithm ID. Maybe we can start with $N$ be Field64 with `PROOFS=2` and $N+1$ be Field64 with `PROOFS=3`? Then allocate more algorithm IDs whenever there is request?

Reply:
yeah, for different choices of `PROOFS`, we'd need to distinct code points (i.e., algorithm IDs)